sklearn/ensemble/bagging.py

```diff
@@ -280,7 +279,10 @@ def _fit(self, X, y, max_samples=None, max_depth=None, sample_weight=None):
         random_state = check_random_state(self.random_state)
 
         # Convert data
-        X, y = check_X_y(X, y, ['csr', 'csc'])
+        X, y = check_X_y(
```

Comment:
Perhaps add a comment that we require X to be 2d and indexable on both axes.

Reply:
Added